### ISSUE TITLE
Inconsistent timeout properties between javadocs and code

### ISSUE THREAD

**Issue by srstsavage** [DataONEorg/d1_libclient_java#1]
Hi, I've been debugging some timeout issues with d1_libclient_java and noticed that there appear to be some incorrect settings and techniques for adjusting the timeout in the javadocs.

## Setting property inconsistencies between javadocs and code

The following comm/grep command shows properties only in the javadocs (column 1), only in code (column 2),
and consistent between the two (column 3).

```
comm <(grep -EIRoh '\* D1Client\.(MNode|D1Node|CNode)\..*$' | sed 's/^* //' | sort -u) <(grep -EIRoh '"D1Client\.(MNode|D1Node|CNode)\.[^"]*"' | tr -d '"' | sort -u)
D1Client.CNode.create.timeout
	D1Client.CNode.create.timeouts
D1Client.CNode.registerSystemMetadata.timeout
	D1Client.CNode.registerSystemMetadata.timeouts
		D1Client.CNode.replication.timeout
		D1Client.CNode.reserveIdentifier.timeout
D1Client.CNode.search.timeout
		D1Client.D1Node.getLogRecords.timeout
		D1Client.D1Node.getSystemMetadata.timeout
		D1Client.D1Node.get.timeout
D1Client.D1Node.listObjects.timeout
		D1Client.MNode.create.timeout
		D1Client.MNode.getReplica.timeout
		D1Client.MNode.replicate.timeout
		D1Client.MNode.update.timeout
```
In other words, `D1Client.CNode.create.timeout` and `D1Client.CNode.registerSystemMetadata.timeout` are mentioned in the javadocs, but `D1Client.CNode.create.timeouts` and `D1Client.CNode.registerSystemMetadata.timeouts` are used in code. The other properties appear to be consistent.

Also, a `setDefaultSoTimeout` method mentioned in javadocs for MultipartCNode, MultipartMNode, MultipartD1Node, MultipartCNode, and MultipartMNode doesn't seem to exist.

**Comment by mbjones** [comment]
Thanks, @shane-axiom this seems like another documentation issue to me. Do you agree?

**Comment by srstsavage** [comment]
Yeah, I agree, mostly the javadocs should just match what's in the code. The only wrinkle here is that for the inconsistent properties, the properties actually used in the code are plural (`timeouts`) and don't match the other timeout properties (`timeout`). Probably not a huge problem but might trip someone up. To fix that there would probably need to be a wrapper method checking both the plural and singular properties since some users might be using the existing plural version.
